pom.xml

```diff
@@ -27,13 +27,13 @@
         <!-- configure Scala version -->
         <scala.major.version>2</scala.major.version>
         <scala.minor.version>13</scala.minor.version>
-        <scala.patch.version>6</scala.patch.version>
+        <scala.patch.version>8</scala.patch.version>
         <scala.binary.version>${scala.major.version}.${scala.minor.version}</scala.binary.version>
         <scala.version>${scala.binary.version}.${scala.patch.version}</scala.version>
         <!-- configure Spark version -->
         <spark.major.version>3</spark.major.version>
-        <spark.minor.version>3</spark.minor.version>
-        <spark.patch.version>4</spark.patch.version>
+        <spark.minor.version>5</spark.minor.version>
+        <spark.patch.version>3</spark.patch.version>
         <spark.binary.version>${spark.major.version}.${spark.minor.version}</spark.binary.version>
         <spark.version>${spark.binary.version}.${spark.patch.version}</spark.version>
```

Comment:
Does the job template support need changing the default Spark versions?

Reply:
I thought the defaults are Spark 3.5.3 and Scala 2.13.8.

I can revert that change, it is nod required.